web/src/left_sidebar_navigation_area.js

```diff
@@ -40,16 +40,22 @@ export function update_dom_with_unread_counts(counts, skip_animations) {
 // TODO: Rewrite how we handle activation of narrows when doing the redesign.
 // We don't want to adjust class for all the buttons when switching narrows.
 
-function remove($elem) {
+function remove_active_class($elem) {
     $elem.removeClass("active-filter active-sub-filter");
 }
 
 export function deselect_top_left_corner_items() {
-    remove($(".top_left_all_messages"));
-    remove($(".top_left_starred_messages"));
-    remove($(".top_left_mentions"));
-    remove($(".top_left_recent_view"));
-    remove($(".top_left_inbox"));
+    const classToRemove = [
```

Comment:
This function name doesn't follow our coding conventions.

Reply:
okay
i will read on zulip coding conventions and fix that